### ISSUE TITLE
Sudden crash of JIRA, Crowd, Confluence

### ISSUE THREAD

**Issue by timea** [Praqma/staci#81]
I was testing the SSO feature between JIRA, Confluence, Crowd when suddenly JIRA was not responding.

<img width="1035" alt="screen shot 2017-02-16 at 09 54 33" src="https://cloud.githubusercontent.com/assets/4135651/23014507/9c26b364-f42f-11e6-8882-b1c79d840d22.png">

Then without me doing anything (no restart of the container, anything), after a few minutes it came back online, but with partially loaded CSS.

<img width="1278" alt="screen shot 2017-02-16 at 10 07 31" src="https://cloud.githubusercontent.com/assets/4135651/23014577/db95155e-f42f-11e6-8042-f045d1efb852.png">

Waiting a bit longer again, it got back to normal.

Now from the logs of JIRA I can see that Crowd has stopped responding.

By this time I am writing, all 3 was down.

While I wrote this sentence, Confluence just came back online.

My CPU load is 15%, but my memory is using 624 MB swap. Could that be the reason?

I have seen this behaviour more times during the course of last week.

**Comment by hoeghh** [comment]
Did you get the logos from crowd? Which version og crowd Are you using? Latest ?

**Comment by timea** [comment]
Crowd Version: 2.9.1 

I found 2 warnings from around the time of the issue. I am just not sure of the exact time, as I found the problem via JIRA not directly.


`2017-02-16 09:27:45,035 http-bio-8095-exec-11 WARN [common.security.jersey.XsrfResourceFilterFactory] Detected usage of the com.atlassian.annotations.security.XsrfProtectionExcluded annotation loaded from elsewhere. com.atlassian.plugins.rest.atlassian-rest-module [29] != WebappClassLoader
  context: /crowd
  delegate: false
  repositories:
    /WEB-INF/classes/
----------> Parent Classloader:
java.net.URLClassLoader@65b54208`


`2017-02-16 09:31:34,154 http-bio-8095-exec-8 WARN [beans.factory.config.CustomEditorConfigurer] Passing PropertyEditor instances into CustomEditorConfigurer is deprecated: use PropertyEditorRegistrars or PropertyEditor class names instead. Offending key [org.codehaus.xfire.service.ServiceFactory; offending editor instance: org.codehaus.xfire.spring.editors.ServiceFactoryEditor@7b21a1ca`

**Comment by hoeghh** [comment]
Might be a ressource problem. The last time we talked the stack ran really slow. Have you checked that crowd has enough memory? The jvm.  I cant remember the setting. 

Does SSO Work?

**Comment by timea** [comment]
Yes, the SSO works between JIRA, Service Desk, Crowd, Confluence and Artifactory. 

I have not done a deep research as we know I am running out of memory. A restart usually helps. Just wanted to let you know in case we conclude it is not a resource issue.
